dirsrvtests/tests/suites/plugins/linked_attributes_test.py

```diff
@@ -0,0 +1,208 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2025 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+import logging
+import pytest
+import ldap
+from lib389.topologies import topology_st
+from lib389._constants import DEFAULT_SUFFIX
+from lib389.plugins import LinkedAttributesPlugin, LinkedAttributesConfigs
+from lib389.idm.user import UserAccounts
+
+pytestmark = pytest.mark.tier1
+
+log = logging.getLogger(__name__)
+
+OU_PEOPLE = f'ou=People,{DEFAULT_SUFFIX}'
+LINKTYPE = 'directReport'
+MANAGEDTYPE = 'manager'
+MANAGER = 'manager1'
+EMPLOYEE = 'employee1'
+INVALID = f'uid=doNotExist,{OU_PEOPLE}'
+
+
+@pytest.fixture(scope='function')
+def manager(topology_st, request):
+    """Fixture to create a manager entry."""
+    log.info('Creating manager entry')
+
+    users = UserAccounts(topology_st.standalone, DEFAULT_SUFFIX)
+    if users.exists(MANAGER):
+        users.get(MANAGER).delete()
+
+    manager = users.create(properties={
+        'uid': MANAGER,
+        'cn': MANAGER,
+        'sn': MANAGER,
+        'uidNumber': '1',
+        'gidNumber': '10',
+        'homeDirectory': f'/home/{MANAGER}'
+    })
+    manager.add('objectclass', 'extensibleObject')
+
+    def fin():
+        log.info('Delete manager entry')
+        if manager.exists():
+            manager.delete()
+
+    request.addfinalizer(fin)
+
+    return manager
+
+
+@pytest.fixture(scope='function')
+def employee(topology_st, request):
+    """Fixture to create an employee entry."""
+    log.info('Creating employee entry')
+
+    users = UserAccounts(topology_st.standalone, DEFAULT_SUFFIX)
+    if users.exists(EMPLOYEE):
+        users.get(EMPLOYEE).delete()
+
+    employee = users.create(properties={
+        'uid': EMPLOYEE,
+        'cn': EMPLOYEE,
+        'sn': EMPLOYEE,
+        'uidNumber': '2',
+        'gidNumber': '20',
+        'homeDirectory': f'/home/{EMPLOYEE}'
+    })
+    employee.add('objectclass', 'extensibleObject')
+
+    def fin():
+        log.info('Delete employee entry')
+        if employee.exists():
+            employee.delete()
+
+    request.addfinalizer(fin)
+
+    return employee
+
+
+@pytest.fixture(scope='function')
+def setup_linked_attributes(topology_st, request):
+    """Fixture to set up the Linked Attributes plugin."""
+    log.info('Setting up Linked Attributes plugin')
+
+    log.info('Enable Dynamic plugins and the Linked Attributes plugin')
+    topology_st.standalone.config.replace('nsslapd-dynamic-plugins', 'on')
```

Comment:
IIRC, we don't support dynamic plugins, and it's mostly for debugging purposes.
I think we'd better test without it (hence, a restart is required here).

Reply:
Thank you, will consult a bit more to see which way is more preferable.

Reply:
@droideck Dynamic plugins are supported, just not enabled by default.